### ISSUE TITLE
Indicate sort order in tables with sortable columns

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2420]
In tables with sort functionality, the currently sorted column should include an `aria-sort` attribute to indicate sort order for assistive tech users. Is this possible in the current pattern since sorting does not happen dynamically, or would that have no bearing?

**Comment by kimisgold** [comment]
All the browse views have a default sort, and yes, `aria-sort` should be applied to the relevant column heading. Right now we're relying on the sort form to indicate the active sort, but I think can apply that attribute and probably a visual indicator to support the form.
